backend/api/serializers.py

```diff
@@ -240,7 +240,7 @@ class Meta:
                   'tags', 'comments', 'votes', 'follower_count',
                   'following_count', 'owned_events_count', 'blocked_users_count',
                   'is_corporate_user', 'corporate_profile')
-        read_only_fields = ('username', 'tags', 'comments', 'votes',
+        read_only_fields = ('id', 'username', 'tags', 'comments', 'votes',
```

Comment:
There is no need to specify `id` as read-only since it's already created as read-only by Django.